sphinx_gallery/gen_rst.py

```diff
@@ -581,6 +581,9 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf):
                     example_rst += code_output
                 else:
                     example_rst += code_output
+                    if not 'sphx-glr-script-out' in code_output:
```

Comment:
I'm a bit confused.
* You want the space after the code stdout, right? (then the not is out of place)
* Or you want it after the image, when there is only the figure an no stdout?(then is ok)